extensions/interactions/MultipleChoiceInput/static/multiple_choice_input.css

```diff
@@ -8,7 +8,8 @@
   color: #0D48A1;
   text-align: left;
   /* This is needed so that images stay bounded by the container in Firefox. */
-  width: 100%;
+  /* Also, width is not fixed. This prevents the accidental selection of option by limiting the selection area to the text and radio button. */
```

Comment:
Comment should be something like:
`This limits the selection area of the option to the text and radio button.`

Reply:
@bansalnitish Thanks!
Making the change.